py/selenium/webdriver/chrome/webdriver.py

```diff
@@ -61,6 +63,7 @@ def __init__(self, executable_path="chromedriver", port=0,
         try:
             RemoteWebDriver.__init__(self,
                 command_executor=self.service.service_url,
+                remote_connection_client=ChromeRemoteConnection,
```

Comment:
remote_connection_client parameter is unnecessary, it should not be passed here and reverted from remote/webdriver.py.

What you need to do is to pass an instance of ChromeRemoteConnection as a value of command_executor parameter.

See the current latest revision for an updated documentation that should make this less confusing:
https://github.com/SeleniumHQ/selenium/blob/a19f07aec61d491fa4839c002d65d03347fc21e8/py/selenium/webdriver/remote/webdriver.py#L64